test/intl402/DateTimeFormat/prototype/format/temporal-objects-format-with-era.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2024 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-datetime-format-functions
+description: >
+  If era option and no other options are provided to DateTimeFormat constructor,
+  objects should be formatted with default options
+features: [Temporal]
+---*/
+
+const formatter = new Intl.DateTimeFormat([], { era: "narrow" });
+
+assert(formatter.format(new Temporal.PlainDate(2025, 11, 4)).startsWith("11"), "formatting a PlainDate should work");
+assert(formatter.format(new Temporal.PlainYearMonth(2025, 11, "gregory")).startsWith("11"), "formatting a PlainYearMonth should work");
+assert(formatter.format(new Temporal.PlainMonthDay(11, 4, "gregory")).startsWith("11"), "formatting a PlainMonthDay should work");
+assert(formatter.format(new Temporal.PlainTime(14, 46)).startsWith("A"), "formatting a PlainTime should work");
+assert(formatter.format(new Temporal.PlainDateTime(2025, 11, 4, 14, 46)).startsWith("11"), "formatting a PlainDateTime should work");
+assert(formatter.format(new Temporal.Instant(0n)).startsWith("12"), "formatting an Instant should work");
```

Comment:
This may fail depending on the host's time zone offset. Probably the best way to work around that is pass `timeZone: 'UTC'` in the formatter options.